css/components/modal.css

```diff
@@ -0,0 +1,34 @@
+.modal {
+  width: 40rem;
```

Comment:
don't set a fixed width of 40 rem, instead set a max-width of 40 rem, otherwise if the content is very small, there would be extra space left useless